README.md

```diff
@@ -65,24 +65,24 @@ Past release notes and **future features planned** is available [here](https://g
 `cleanlab` is:
 
 1.  **backed-by-theory** - Provable perfect label error finding in some realistic conditions.
-2.  **fast** - Non-iterative, parallelized algorithms (e.g. < 1 second to find label errors in ImageNet with pre-computed probabilities)
+2.  **fast** - Non-iterative, parallelized algorithms (e.g. < 1 second to find label issues in ImageNet with pre-computed probabilities)
 3.  **general** - Works with any dataset, model, and framework, e.g., Tensorflow, PyTorch, sklearn, xgboost, etc.
 
-## Find label errors with PyTorch, Tensorflow, sklearn, xgboost, etc. in 1 line of code
+## Find confident_learning with PyTorch, Tensorflow, sklearn, xgboost, etc. in 1 line of code
 
 ```python
 # Compute psx (n x m matrix of predicted probabilities) on your own, with any classifier.
 # Here is an example that shows in detail how to compute psx on CIFAR-10:
 #    https://github.com/cleanlab/examples/tree/master/cifar10
 # Be sure you compute probs in a holdout/out-of-sample manner (e.g. via cross-validation)
-# Now getting label errors is trivial with cleanlab... its one line of code.
-# Label errors are ordered by likelihood of being an error. First index is most likely error.
-from cleanlab.pruning import get_noise_indices
+# Now getting confident_learning is trivial with cleanlab... its one line of code.
```

Comment:
confident_learning -> label issues

Reply:
this substitution error seems to recur, so worth checking every occurrence of "confident_learning" in the new code.

Reply:
and every occurrence of "labels"

Reply:
Ran a search and only found these two cases for `confident_learning`. Also, ran a similar search / fix for the `labels labels` issue.